src/index.html

```diff
@@ -11,6 +11,20 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form class="form" data-qa="big">
+      <input
+          class="form__input form__input--large"
+          type="text"
+          placeholder="Try &quot;Los Angeles&quot;"
+          data-qa="keypress"
+        >
```

Comment:
```suggestion
      <input
        class="form__input form__input--large"
        type="text"
        placeholder="Try &quot;Los Angeles&quot;"
        data-qa="keypress"
      >
```
Pay attention to the formatting